packages/webview_flutter/webview_flutter_wkwebview/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## NEXT
+## 2.9.6
```

Comment:
Can we not leave these all as NEXT?

Reply:
We could potentially leave a lot of them as `NEXT`, but our standard policy is to publish `example/lib/main.dart` changes, and a lot of the packages touched here have changes there. And since template-created projects use `flutter_lints` now, which includes `avoid_print`, it's kind of nice to have the example code not show lint errors if people copy/paste the entire thing into a project as a starting point.